src/HealthChecks.UI/ServiceCollectionExtensions.cs

```diff
@@ -114,6 +116,8 @@ public static IServiceCollection AddWebhooksEndpointHttpClient(this IServiceColl
             .Services;
         }
 
+        private static bool databaseDeleted = false;

```

Comment:
```suggestion
        private static bool databaseDeleted;

```

Reply:
Thank you very much! I didn't realize of it. Also renamed the variable